helm-chart/flink-job-cluster/values.yaml

```diff
@@ -0,0 +1,113 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: flink
+  tag: 1.9.3
+
+# Prometheus reporter jar to be loaded by flink
+envVars:
+  - name: HADOOP_CLASSPATH
+    value: /opt/flink/opt/flink-metrics-prometheus-1.9.3.jar
+
+jobManager:
+  accessScope: Cluster
+  ports:
+    ui: 8081
+
+# enable metrics ports for jobManager  
+  metrics: 
+    enabled: true
+    extraPorts:
+      - name: prom
+        containerPort: 9249
+
+  resources:
+    limits:
+      memory: 1024Mi
+      cpu: 200m
+
+taskManager:
+  replicas: 2
+  
+  volumeMounts: {}
+
+# enable metrics ports for taskManager
+  metrics:
+    enabled: true
+    extraPorts:
+      - name: prom
+        containerPort: 9249
+        protocol: TCP
+
+  resources:
+    limits:
+      memory: 1024Mi
+      cpu: "200m"
+
+job:
+  # job will look for a JAR file at /JARFiles/<JAR_FILE> and execute it
+  # className has to be valid and used in the provided JAR File
+  jarFile: /JARfiles/JARfile.jar
```

Comment:
Can we replace this with `./examples/streaming/WordCount.jar` which exists in the container, so that the example could run successfully?

Reply:
Sure! I've replaced it in the user guide and values.yaml, the result is now better:
```
NAME                                                READY   STATUS    RESTARTS   AGE
flink-job-cluster-job-mnxl4                         1/1     Running   0          17s
flink-job-cluster-jobmanager-85cc48fdc9-mfpvt       1/1     Running   0          59s
flink-job-cluster-taskmanager-7787cf998c-zstrj      1/1     Running   0          59s
flink-job-cluster-taskmanager-7787cf998c-zz4hh      1/1     Running   0          59s
flink-operator-controller-manager-9676cfbcf-86565   2/2     Running   0          4d21h
```